src/kitovu/cli.py

```diff
@@ -62,3 +66,51 @@ def validate(config: typing.Optional[pathlib.Path] = None) -> None:
         syncing.validate_config(config, CliReporter())
     except utils.UsageError as ex:
         raise click.ClickException(str(ex))
+
+
+@cli.command()
+def docs() -> None:
+    """Open the documentation in the browser."""
+    # FIXME: make version aware
+    webbrowser.open_new_tab('https://kitovu.readthedocs.io/en/latest')
+
+
+DEFAULT_EDITORS = [
+    'vim',
+    'emacs',
+    'nano',
+    'editor',
+    'notepad',
+]
+
+
+@cli.command()
+@click.option('--config', type=pathlib.Path, help="The configuration file to edit")
+@click.option('--editor', type=str, help=f"The command of the editor to use. \
+              Default: $EDITOR or the first existing out of {', '.join(DEFAULT_EDITORS)}")
+def edit(config: typing.Optional[pathlib.Path] = None, editor: typing.Optional[str] = None) -> None:
+    """Edit the specified configuration file."""
+    if editor is None and 'EDITOR' in os.environ:
+        editor = os.environ['EDITOR']
+    editor_path: typing.Optional[str] = _get_editor_path(editor)
+
+    if editor_path is None:
+        if editor is None:
+            print('Could not find a valid editor')
+        else:
+            print(f"Could not find the editor {editor}")
+        sys.exit(1)
+
+    subprocess.call([editor_path, config])
```

Comment:
mypy complains here because it thinks `editor_path` can still be `None`. You should be able to fix it by adding a `assert editor_path is not None` above (I hope).

Reply:
Adding the default value fixed it. I've no idea why I thought this would lead to a circular dependency. Maybe because I think the cli uses too many namespaces and if we would want to extract the `get_config_file_path` to prevent it, it would lead to circular dependencies.